dist/pythonlibs/riotctrl_shell/ieee802154.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (C) 2019-20 Freie Universität Berlin
```

Comment:
Please move this whole file content to your application's test script, as it only implements shell interactions for application-specfic shell-commands.

Reply:
Sorry did not reallize those are from the Release specs (see https://github.com/RIOT-OS/Release-Specs/pull/222).